packages/@react-aria/tabs/src/useTabs.ts

```diff
@@ -55,11 +54,6 @@ export function useTabs<T>(props: TabsAriaProps<T>, state: SingleSelectListState
     disallowEmptySelection: true
   });
 
-  // Ensure a tab is always selected
```

Comment:
I figured it would be nice to allow users to customize their own tab fallback strategy when a tab isn't selected or the selected tab is deleted from the child list so I pulled it out to the spectrum package